### ISSUE TITLE
.vue errors not reported in terminal

### ISSUE THREAD

**Issue by braincomb** [nabla/vite-plugin-eslint#11]
Hi there!

I've noticed that this plugin is not reporting `.vue` eslint errors/warnings in the terminal. My `vite.config.js` looks as following:

```js
import { defineConfig } from "vite";
import vue from "@vitejs/plugin-vue";
import eslintPlugin from "@nabla/vite-plugin-eslint";
import path from "path";

export default defineConfig({
  plugins: [vue(), eslintPlugin({ eslintOptions: { cache: false } })],
  resolve: {
    alias: {
      "@": path.resolve(__dirname, "./src"),
    },
  }
});
```

There's no issue with `.js` or `.ts` files, just `.vue` files that are not being reported on. ESLint warnings are also correctly reported inside my `.vue` files in the code, and in the Problems tab in VSCode, just not the terminal.

If I'm reading the `path` option regexp correctly, then `.vue` files should be on by default. So curious if possibly a bug or if I'm doing something wrong?

Thanks!

**Comment by braincomb** [comment]
Seems to work now after reloading the dev server. 👍
